vrecord.rb

```diff
@@ -35,6 +36,20 @@ def caveats
     end
   end
 
+  if MacOS.version < :mojave
+    def caveats
+      <<~EOS
+        ** IMPORTANT FOR macOS INSTALL **
+        Additional install steps are necessary for a fully functioning Vrecord
+        install on macOS. Vrecord uses mkvtoolnix for some optional steps such
+        as embedding logs into Matroska files. mkvtoolnix is automatically
+        installed via homebrew if the Mac OS is Mojave or greater, else we
+        recommend that mkvtoolnix is installed via
+        https://mkvtoolnix.download/macos/MKVToolNix-46.0.0.dmg
```

Comment:
Is there any reason to choose 46? The current one is 49. I would prefer a more generic link, such as `https://mkvtoolnix.download/macos/`, indicating to chose the last available release.

Reply:
I note that at https://mkvtoolnix.download/downloads.html#macosx it says

> Up to and including MKVToolNix v41, the DMGs worked on at least 10.12 "Sierra" and newer. 

Does anything newer than v41 work on 10.12 - or would it be worth pointing to that version?

Reply:
>Does anything newer than v41 work on 10.12 - or would it be worth pointing to that version?

Hmm… I didn’t notice that. Thank you for the information! Do you have any clue why? The “C++17 issue”, which appeared in version 47, can be resolved with a small patch, as I pointed out earlier.

Reply:
Absolutely no idea why! (Or if that is even accurate). I have no way to test unfortunately

Reply:
No problem. I’ll try to find the time this weekend to play a little with old compilers… usually they work best with old software.